lib/explorer/data_frame.ex

```diff
@@ -54,7 +54,7 @@ defmodule Explorer.DataFrame do
   ## Options
 
     * `delimiter` - A single character used to separate fields within a record. (default: `","`)
-    * `dtypes` - A keyword list of `[column_name: dtype]`. If `nil`, dtypes are imputed from the first 1000 rows. (default: `nil`)
+    * `dtypes` - A list of `{"column_name", dtype}` tuples. Uses column names as read, not as defined in options. If `nil`, dtypes are imputed from the first 1000 rows. (default: `nil`)
```

Comment:
Just to throw out there, cause I am not sure it is a good idea, you could also make this be: `dtypes: [string: "foo", string: "bar", integer: "baz"]`.

Reply:
I personally like the 2-tuples better